rest-api/src/main/resources/paths/index.yml

```diff
@@ -94,6 +94,8 @@
     $ref: ./apps/v1_apps_appId_participants_userId.yml
 /v1/apps/{appId}/participants/healthCode:{healthCode}:
     $ref: ./apps/v1_apps_appId_participants_healthCode.yml
+/v1/apps/{appId}/participants/healthCode:{healthCode}/versions/{version}:
+    $ref: ./apps/v1_apps_appId_participants_healthCode_versions_version.yml
```

Comment:
Any utility in calling this "userId" but then parsing it so it can be health code, userId, external ID, etc?

Reply:
Deleted.